components/raftstore/src/store/fsm/store.rs

```diff
@@ -117,6 +120,9 @@ pub const PENDING_MSG_CAP: usize = 100;
 pub const ENTRY_CACHE_EVICT_TICK_DURATION: Duration = Duration::from_secs(1);
 pub const MULTI_FILES_SNAPSHOT_FEATURE: Feature = Feature::require(6, 1, 0); // it only makes sense for large region
 
+const PERIODIC_FULL_COMPACT_TICK_INTERVAL_DURATION: Duration = Duration::from_secs(30 * 60); // Check every half hour.
+const PERIODIC_FULL_COMPACT_CPU_MAX_USAGE_PCT: f64 = 0.10;
```

Comment:
this may be too aggressive. thoughts on what to change it to? make it configurable?

Reply:
made this configurable.